### DISCUSSION TITLE
Add support for Reimbursement accounts

### DISCUSSION THREAD

**Discussion by pashagolub** [ananthakumaran/paisa#88] [Ideas]
[Tracking reimbursable expenses](https://ledger-cli.org/doc/ledger3.html#Tracking-reimbursable-expenses-1):
> Sometimes you will want to spend money on behalf of someone else, which will eventually get repaid. Since the money is still yours, it is really an asset. And since the expenditure was for someone else, you don’t want it contaminating your Expenses reports. You will need to keep an account for tracking reimbursements.

> This is fairly easy to do in ledger. When spending the money, spend it to your Assets:Reimbursements, using a different account for each person or business that you spend money for. For example:
```
2004/09/29  Circuit City
    Assets:Reimbursements:Company XYZ     $100.00
    Liabilities:MasterCard
```

For example all my business trips expenses are going under Reimbursement. I use several Reimbursement accounts and they shown as Investments in paisa right now. Which is not the case really.
 
I would exclude Reimbursement accounts from Investments dashboards. Maybe later we can add a separate dashboard to track reimbursable expenses.

**Comment by ananthakumaran** [comment]
I converted this to a discussion since this is a feature request, not a bug.
